client/analytics/settings/historical-data/index.js

```diff
@@ -2,90 +2,84 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { addQueryArgs } from '@wordpress/url';
-import apiFetch from '@wordpress/api-fetch';
 import { Component } from '@wordpress/element';
 import { compose } from '@wordpress/compose';
-import moment from 'moment';
-import { NOTES_STORE_NAME } from '@woocommerce/data';
-import { withDispatch } from '@wordpress/data';
+import {
+	IMPORT_STORE_NAME,
+	NOTES_STORE_NAME,
+	QUERY_DEFAULTS,
+} from '@woocommerce/data';
+import { withDispatch, withSelect } from '@wordpress/data';
 import { withSpokenMessages } from '@wordpress/components';
-import { recordEvent } from '@woocommerce/tracks';
+import { SECOND } from '@fresh-data/framework';
 
 /**
  * Internal dependencies
  */
 import { formatParams } from './utils';
 import HistoricalDataLayout from './layout';
-import { QUERY_DEFAULTS } from '../../../wc-api/constants';
-import withSelect from '../../../wc-api/with-select';
 
 class HistoricalData extends Component {
 	constructor() {
 		super( ...arguments );
 
 		this.dateFormat = __( 'MM/DD/YYYY', 'woocommerce-admin' );
+		this.intervalId = -1;
+		this.lastImportStopTimestamp = 0;
+		this.cacheNeedsClearning = true;
```

Comment:
typo

Reply:
Good eyes @becdetat! I corrected it in the commit 2637750